doc/content/examples/divertor_monoblock/index.md

```diff
@@ -0,0 +1,251 @@
+# Divertor Monoblock During Pulsed Operation
+
+TMAP8 is used to model tritium transport in a divertor monoblock to elucidate the effects of pulsed operation (up to fifty 1600 s plasma discharge and cool-down cycles) on the tritium in vessel inventory source term and ex-vessel release term (i.e., tritium retention and permeation) for safety analysis. This example reproduces the results presented in [!cite](Shimada2024114438).
```

Comment:
```suggestion
TMAP8 is used to model tritium transport in a divertor monoblock to elucidate the effects of pulsed operation (up to fifty 1600-second plasma discharge and cool-down cycles) on the tritium in-vessel inventory source term and ex-vessel release term (i.e., tritium retention and permeation) for safety analysis. This example reproduces the results presented in [!cite](Shimada2024114438).
```

Would prefer spelling out "second" here. Since we're spelling it out and using it as an adjective, we must add the hyphen for correctness.